lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -422,6 +423,20 @@ collectOpsToPipeline(scf::ForOp forOp,
     opInfo[loadOp] = loadInfo;
   };
 
+  // Last, find load users and assigning a stage for them.
+  // We cannot use forOp.walk(...) here because we only want to visit the
+  // operations in the loop body block. Nested blocks are handled separately.
+  for (Operation &op : forOp) {
+    auto I = opInfo.find(&op);
```

Comment:
What does `I` mean? If it's `info`, better to spell it out I think.

Reply:
It's short for "Iter". Does iter sound better to you?

Reply:
Yeah